matching-service/API.md

```diff
@@ -0,0 +1,85 @@
+# Matching Service
+
+## APIs
+
+### 1. Getting an Active Match
+
+```http
+GET /match/:username
+```
+
+| Parameter  | Type     | Description                    |
+| :--------- | :------- | :----------------------------- |
+| `username` | `string` | **Required**. Current username |
+
+### Responses
+
+```javascript
+{
+    "_id": string,
+    "matchId": string,
+    "username1": string,
+    "username2": string,
+    "isActive": true,
+    "__v": 0
+}
+
+OR
+
+{
+    message: 'Could not find an existing match!'
+}
+```
+
+If the current user has an active match, the reponse will contain the match object with the relavant infomation.
```

Comment:
typo here: 'reponse' → 'response', 'relavant' → 'relevant'. same for PUT request below

Reply:
thanks for spotting this